source/fem-dem/vans_assemblers.cc

```diff
@@ -1717,10 +1722,9 @@ VANSAssemblerViscousTorque<dim>::calculate_particle_fluid_interactions(
     RequiresConstantDensity(
       "VANSAssemblerViscousTorque<dim>::calculate_particle_fluid_interactions"));
 
-  const double density = scratch_data.properties_manager.get_density_scale();
-  const double kinematic_viscosity =
-    scratch_data.properties_manager.get_kinematic_viscosity_scale();
-
+  const std::vector<double> density = scratch_data.density_at_particle_location;
+  const std::vector<double> kinematic_viscosity =
```

Comment:
```suggestion
  const std::vector<double>& density = scratch_data.density_at_particle_location;
  const std::vector<double>& kinematic_viscosity =
```
Should be references to avoid copies.